driver/setup.py

```diff
@@ -7,7 +7,7 @@
 setup_requirements: List[str] = []
 
 requirements = ["pace-util", "fv3core", "fv3gfs-physics", "pace-stencils", "dacite"]
-if sys.version_info.major == 3 and sys.version_info.minor == 6:
+if sys.version_info.major == 3 and sys.version_info.minor == 8:
     requirements.append("dataclasses")
```

Comment:
```suggestion
```
You can delete this clause entirely, it only existed to provide support for 3.6.